app/assets/javascripts/angular/controllers/mobile/messages.js

```diff
@@ -0,0 +1,197 @@
+(function() {
+  // Set CSRF Headers to be used for Submitting Messages via Ajax.
+  angular.module('wtsApp').config([
+    "$httpProvider", function($httpProvider) {
+      $httpProvider.defaults.headers.common['X-CSRF-Token'] = $('meta[name=csrf-token]').attr('content');
+    }
+  ]);
+
+  angular.module('wtsApp').controller('messagesCtrl', ['$scope', '$http', '$interval', function($scope, $http, $interval) {
+    // Default state for Messages.
+    $scope.initializeMessages = function() {
```

Comment:
These functions here are for setting the options for filtering messages, setting what teams you are allowed to send messages too, selecting a conversation, resetting the new message form after a message has been sent, etc.